submissions/fabunmiTolulope/html-forms/contact-form.html

```diff
@@ -13,8 +13,8 @@ <h1>Welcome to Greenprint Integrated Services</h1>
         <ul>
                 <li><a href="#"> Home</a></li>
                 <li><a href="#">Blog</a> </li>
-                <li><a href="../HTML/Productpage.html">Productpage</a> </li>
```

Comment:
Why are you removing the links?

Reply:
@adaugochi the links I removed was only for the images,  I had them in a folder called images in the css folder and didn't want you to complain about it..
Then I don't understand that line 16 you highlighted.. thanks

Reply:
oooh sorry my bad, I felt we were not told to link it together that was why I removed the links.

Reply:
@adaugochi i have added the navigation links.